.github/workflows/github-pages.yml

```diff
@@ -53,7 +53,6 @@ jobs:
 
       - name: Deploy To Pages
         uses: peaceiris/actions-gh-pages@v3           # see https://github.com/peaceiris/actions-gh-pages for details
-        if: ${{ github.ref == 'refs/heads/release' }} # set to 'refs/heads/release' to only deploy when the branch is release
```

Comment:
I don't have enough context for this line, just want to make sure it's removed with a purpose